### PR TITLE
Update to new documentation layout

### PR DESCRIPTION
**Pull request by bartaz**
## Done

Updates docs to use new layout from Vanilla 4.5.

## QA

- Review docs in new layout, make sure it works as expected
  - https://microk8s-io-624.demos.haus/docs

image

### CODE REVIEW

File: templates/base_layout.html [hunk 1]
```diff
@@ -80,24 +80,25 @@
   <link rel="stylesheet" type="text/css" media="screen" href="{{ versioned_static('css/styles.css') }}">
 </head>
 
-<body class="no-js is-paper l-site">
+
```
Comment (bartaz) [inline templates/base_layout.html:83]: This is a bit of refactoring that is required to make base layout support standard site layout and new docs layout.

File: templates/partial/_navigation.html [hunk 2]
```diff
@@ -1,44 +1,68 @@
...
-  <div class="row">
-    <div class="p-navigation__banner col-3 u-no-padding--left">
-      <div class="p-navigation__tagged-logo">
-        <a class="p-navigation__link" href="/">
-          <div class="p-navigation__logo-tag">
-            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="Canonical" width="95">
-          </div>
-          <span class="p-navigation__logo-title">MicroK8s</span>
-        </a>
+  {% if is_docs %}
```
Comment (bartaz) [inline templates/partial/_navigation.html:46]: This partial needs to support 2 variants of top navigation - standard site one, and full width version using in docs layout.

Shared parts are extracted to macros, so they can be reused in both versions.

File: templates/partial/_footer.html [hunk 3]
```diff
@@ -1,3 +1,34 @@
+{% if is_docs %}
```
Comment (bartaz) [inline templates/partial/_footer.html:1]: Expanding footer partial to support 2 versions of it - one for standard site, another for full with docs layout.

**Review by andesol — APPROVED** [review]

**Comment by lyubomir-popov** [comment]
@bartaz just one thing: ther'es a slight misalignment, can we fix it?
image

**Pull request merged**